CI/deploy_pypi.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
can we automate running this script in Travis or Github actions every time a tag is created?

Reply:
Yes, we should do that, let me try to add that to the Travis config.
By the way, this script requires a PyPI API token.
Currently, I am using mine to upload to the PyPI (https://pypi.org/project/p4runtime/) and I own this package.
I can move the ownership to anyone and add the token for Travis bot to the PyPI

Reply:
I think Travis has a mechanism to encrypt secrets / tokens, feel free to keep using yours unless this is an issue for you.

If you could add me as a maintainer to the pypi project - AntoninBas (`antonin.bas[at]gmail.com`) - in case some changes are required in the future, it would be great. Thanks.

Reply:
Is this yours? (I need to use the username to invite users to the project)
https://pypi.org/user/AntoninBas/

Reply:
yes that's me